### DISCUSSION TITLE
文档结尾 上一篇 下一篇 配置

### DISCUSSION THREAD

**Discussion by qmmm61** [volantis-x/hexo-theme-volantis#687] [Q&A] [answered]
想要不显示文档结尾上一篇下一篇组件，查阅文档没找到对应设置，看到有[issue#425](https://github.com/volantis-x/hexo-theme-volantis/issues/425)提及过，并且issue中表示已经完成该功能，可是没有详细描述如何设置。

**Comment by inkss** [comment]
这不得问 @xaoxuu 呀

**Comment by xaoxuu** [comment, marked as answer]
不支持关闭
